build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
Use `#!/usr/bin/env bash` shebang instead. Should not have this one in any aspnet repo.